### PR TITLE
#2 - added test for install

### LINKED ISSUE #2: IT for install
Let's create IT for `apt-get install` to make sure package can be installed from debian-adapter.

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #2 
Added test for install.

### CODE REVIEW

File: src/test/java/com/artipie/debian/DebianSliceITCase.java [hunk 1]
```diff
@@ -118,6 +121,27 @@ void searchWorks() throws IOException, InterruptedException {
...
+        MatcherAssert.assertThat(
+            "Configuration was delayed",
+            res.getStderr(),
+            new StringContainsInOrder(
+                new ListOf<>(
+                    "debconf: delaying package configuration",
+                    "since apt-utils is not installed"
+                )
+            )
+        );
```
Comment (olegmoz) [inline src/test/java/com/artipie/debian/DebianSliceITCase.java:142]: @olenagerasimova could you please explain why this check is required?

**Review by olegmoz — CHANGES_REQUESTED** [review]

File: src/test/java/com/artipie/debian/DebianSliceITCase.java (hunk 1 above)
Comment (olenagerasimova) [reply src/test/java/com/artipie/debian/DebianSliceITCase.java:142]: @olegmoz let me explain the situation: I failed to find a package without any dependencies at all or which can be fully installed without any settings. Installation of this one is actually finished with this error, but as the package was successfully downloaded and unpacked this error does not actually matter to us. This check is for the sake of the test, we have to be sure that the error is exactly this one, not any other.

Comment (olegmoz) [reply src/test/java/com/artipie/debian/DebianSliceITCase.java:142]: @olenagerasimova it looks more like here we ensure that this error occurred. There is no check that no other errors occurred: if there are other info in output the test will pass. And if package is changed to one without dependencies the test will fail, which is bad. I'd suggest to remove this assertion and add a `@todo` for finding a package without dependencies or creating one

**Comment by olenagerasimova** [comment]
@olegmoz thanks, added todo and removed assertion

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, good to merge

**Pull request merged**